spec/index.html

```diff
@@ -421,6 +569,97 @@ <h2>Fragment identifiers</h2>
       <h3>Examples</h3>
       <p class="ednote">FIXME</p>
     </section>
+    <section id="faq" class="informative" data-format="markdown">
+      <p class="ednote">REMOVE THIS SECTION BEFORE PUBLICATION.</p>
```

Comment:
This section should pop out to be Appendix B IMO. Otherwise, it seems to be part of IANA Considerations.